tests/pages/HomePage.test.tsx

```diff
@@ -1,24 +1,14 @@
-import { mount } from "enzyme";
```

Comment:
Maybe add a check for the background?

Reply:
I spent a lot of time this morning trying to figuring out how we could test that, because it's not really a background (in term of css) but a svg that takes full width.
Then, I read some interesting things about components testing and the philosophy behind `testing-library` (cf [here](https://kentcdodds.com/blog/testing-implementation-details) ) and I ended up thinking that testing this bg presence was more about implementation details. From that, I decided to test if there was the provider name & the link to connect displayed, since this is the kind of thing the user will see(k). This reasoning is also motivated by the fact that these two points were less change-sensitive and would cause less test refacto, instead of the BG implementation (some client could decide to put this bg using css and not a svg). I hope my thought seems clear to you 😅

Reply:
That's a very good point :). Next question then:
Do we need this test then? We are already testing the link in the e2e tests, so all it really does it to check if we have `Fewlines` written.

Reply:
I don't have answer for this question right now. The limit is hard to find. In this case, almost every component tests is useless, so I don't know what to think.

Reply:
We probably should have a discussion about this then. There is no real answer, we just need to decide if we heavily rely on e2e tests, on split them with unit/integration tests.